### DISCUSSION TITLE
Scope param of Security does not evaluate F-strings

### DISCUSSION THREAD

**Discussion by item1a** [fastapi/fastapi#11302] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Sequence, Optional

# third party imports
from fastapi import APIRouter, Path, Query, Body, Depends, Security

# module imports
from logic.image import ImageLogic
from api.dependencies import image_logic_dependency, authorize_user
from models.image import Image, ImageCreate, ImageUpdate
from models.response import AddResponse, UpdateResponse, DeleteResponse
from enum import Enum

router = APIRouter()
class CRUDOperation(str, Enum):
    CREATE = 'create'
    READ = 'read'
    UPDATE = 'update'
    DELETE = 'delete'

class ResourceType(str, Enum):
    USER = 'user'
    IMAGE = 'image'
    USER_IMAGE = 'user_image'
    
@router.post('', response_model=AddResponse)
async def create(auth_info: str = Security(authorize_user, scopes=f"{CRUDOperation.CREATE}:{ResourceType.IMAGE}"),
                  image_logic: ImageLogic = Depends(image_logic_dependency), image: ImageCreate = Body(..., title='create image')):

    user_email = auth_info
    added = await image_logic.create(image=image, user_email=user_email)
    return AddResponse(added=added)
```

### Description

`scopes=f"{CRUDOperation.CREATE}:{ResourceType.IMAGE}"` is returning `{CRUDOperation.CREATE}:{ResourceType.IMAGE}`, not `create:image`

### Operating System

Windows

### FastAPI Version

0.110.0

### Pydantic Version

2.6.4

### Python Version

3.12.2

**Comment by JavierSanchezCastro** [comment]
Do you have an MRE?

**Comment by YuriiMotov** [comment, marked as answer]
```
f"{CRUDOperation.CREATE.value}:{ResourceType.IMAGE.value}"
```

**Reply by item1a** [reply]
thank you very much
